consumer-rest-core/src/main/scala/za/co/absa/spline/consumer/rest/controller/ExecutionEventsController.scala

```diff
@@ -53,7 +54,11 @@ class ExecutionEventController @Autowired()(val repo: ExecutionEventRepository)
     @ApiParam(value = "Sort Direction", example = "asc")
```

Comment:
Not sure about this name. Maybe it is just me but I never heard it be called like that. Usually, sort order/ordering/etc.

Reply:
The term _sort direction_ is used, although less often than the others, I agree.
For example Spark and certain Angular components use this term, that I think is why it made it to Spline API. Nevertheless I renamed it to `sortOrder`